floss/main.py

```diff
@@ -127,25 +111,25 @@ def get_all_plugins():
     """
     ps = DecodingRoutineIdentifier.implementors()
     if len(ps) == 0:
-        ps.append(plugins.function_meta_data_plugin.FunctionCrossReferencesToPlugin())
-        ps.append(plugins.function_meta_data_plugin.FunctionArgumentCountPlugin())
-        ps.append(plugins.function_meta_data_plugin.FunctionIsThunkPlugin())
-        ps.append(plugins.function_meta_data_plugin.FunctionBlockCountPlugin())
-        ps.append(plugins.function_meta_data_plugin.FunctionInstructionCountPlugin())
-        ps.append(plugins.function_meta_data_plugin.FunctionSizePlugin())
-        ps.append(plugins.function_meta_data_plugin.FunctionRecursivePlugin())
-        ps.append(plugins.library_function_plugin.FunctionIsLibraryPlugin())
-        ps.append(plugins.arithmetic_plugin.XORPlugin())
-        ps.append(plugins.arithmetic_plugin.ShiftPlugin())
-        ps.append(plugins.mov_plugin.MovPlugin())
+        ps.append(function_meta_data_plugin.FunctionCrossReferencesToPlugin())
+        ps.append(function_meta_data_plugin.FunctionArgumentCountPlugin())
+        ps.append(function_meta_data_plugin.FunctionIsThunkPlugin())
+        ps.append(function_meta_data_plugin.FunctionBlockCountPlugin())
+        ps.append(function_meta_data_plugin.FunctionInstructionCountPlugin())
+        ps.append(function_meta_data_plugin.FunctionSizePlugin())
+        ps.append(function_meta_data_plugin.FunctionRecursivePlugin())
+        ps.append(library_function_plugin.FunctionIsLibraryPlugin())
+        ps.append(arithmetic_plugin.XORPlugin())
+        ps.append(arithmetic_plugin.ShiftPlugin())
+        ps.append(mov_plugin.MovPlugin())
```

Comment:
these changes are valid syntactically, so no problem there.

for this project, since we have so many imports, we've been keeping the module prefix on most elements so that its clear where code/objects are coming from. all things being equal, let's keep it this way for consistency.